### DISCUSSION TITLE
How many CFs can we control by using crazyswarm

### DISCUSSION THREAD

**Discussion by imrd1123** [USC-ACTLab/crazyswarm#535] [Q&A] [answered]
Thank you for your help with my previous question.

I am looking to increase the number of CF(Crazyflie) to control (I am currently using 6).
How many CFs can we control at maximum using the current crazyswarm system?
(I know from reading crazyswarm's document that he has successfully flown 49 CFs with 3 crazyradios.)

If you know, I would appreciate it if you could let me know.

**Comment by jpreiss** [comment, marked as answer]
To my knowledge, 49 is still the most anyone has tried. It should be possible to control more than 49 by adding radios or reducing the motion capture frequency.

**Reply by imrd1123** [reply]
Thank you for your reply.
I understood that there were no examples of flying more than 49 cfs. 
So the performance of the motion capture and the number of Crazyflieradio communications may be a factor. 
I'm still considering flying more than 49 cfs, but I will take this advice into account when I do.
